### ISSUE TITLE
Invalid Hex Conversion

### ISSUE THREAD

**Issue by allisteraligerum** [aligerum/umbriel#24]
For some reason, hex conversion isn't working properly for all colors.

```coffee
color = new Color 255/2, 255/2, 255/2
console.log color.format "hex"   #outputs: "7f.87f.87f.8"
console.log color.format "hex8"  #outputs: "7f.87f.87f.8ff"
```

**Comment by allisteraligerum** [comment]
Appears to be related to floats; integers work fine.

**Comment by allisteraligerum** [comment]
Fixed by commit 598e6d0c2d196bbf9764620ea295e38fbd7c74eb.
